_includes/setup/android-setup-linux.md

```diff
@@ -0,0 +1,88 @@
+## Android setup
+
+_This step is optional, and can be performed at any time before
+using Flutter to develop apps for Android._
+
+### Install Android Studio
+
+To develop Flutter apps for Android, you need to install Android Studio.
+
+1. Install [Android Studio](https://developer.android.com/studio/index.html).
+
+   Flutter requires the latest Android SDK and Android SDK Platform-Tools packages.
+   You can verify these are up-to-date in **Android Studio>Configure>SDK Manager**,
+   as described in [Update the IDE and SDK Tools](https://developer.android.com/studio/intro/update.html).
+
+1. Start Android Studio.
+
+### Set up your Android device
+
+To prepare to run and test your Flutter app on Android, you'll need an Android device running
+Android 4.1 (API level 16) or higher.
+
+1. Enable developer mode on your device by visiting **`Settings>About phone`** and
+tapping the **Build number** line seven times.
+2. In **Settings>Developer options**, enable **USB debugging**.
+3. Using a USB cable, plug your phone into your computer. If prompted on your
+device, authorize your computer to access your device.
+4. In the terminal, run the `flutter devices` command to verify that Flutter recognizes your
+connected Android device.
+5. Start your app by running `flutter run`.
+
+By default, Flutter uses the version of the Android SDK where your `adb` tool is based. If
+you want Flutter to use a different installation of the Android SDK, you must set the
+`ANDROID_HOME` environment variable to that installation directory.
+
+### Set up the Android emulator
+
+To prepare to run and test your Flutter app on the Android emulator, follow these steps:
+1. Enable Kernel-based Virtual Machine (KVM) support on your machine.
+1. Define a virtual device in Android Virtual Device Manager.
+
+#### Enable KVM
```

Comment:
Thanks for the link! That's a really good page.... FYI one of the things that's going on in this PR is I'm testing everything that's possible for me to test. And this procedure 1. does not work on Ubuntu workstation b/c chip doesn't support it, 2. just works on corp Mac, and 3. can't test on Windows. What I notice is, if the steps were already vetted, these types of comments would be minimal. Is someone on the team responsible for testing the setup procedure and its instructions?

If the broader question is, what's the thinking behind having a very long procedure here, the answer is it is a best practice not to bounce users around to various websites during setup. They get lost and/or frustrated. The remedy for long documentation is usually product-related changes upstream (though we're not in control of the KVM steps in this particular example).

For very specific feedback where you feel what's there is actually wrong, it's totally fine to edit the PR directly or submit a separate PR. I'll take a stab at this, but if you have further feedback on it, let's go the direct route, OK?